sklearn/cluster/hierarchical.py

```diff
@@ -956,3 +965,12 @@ def fit(self, X, y=None, **params):
     @property
     def fit_predict(self):
         raise AttributeError
+
+    @property
```

Comment:
It's fine to inherit this from `AgglomerativeClustering` and not define it again.

Reply:
Thanks, this should be solved by commit #238c4ac